packages/gp2-components/src/organisms/ProjectDetailHeader.tsx

```diff
@@ -0,0 +1,114 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import {
+  BackLink,
+  crossQuery,
+  drawerQuery,
+  ExternalLink,
+  pixels,
+  Subtitle,
+  utils,
+} from '@asap-hub/react-components';
+
+import { css } from '@emotion/react';
+
+import { format, formatDistanceStrict as formatDistance } from 'date-fns';
+import colors from '../templates/colors';
+
+import usersIcon from '../icons/users-icon';
+import dateIcon from '../icons/date-icon';
+import { projectsImage } from '../images';
+import CardWithBackground from '../molecules/CardWithBackground';
+import IconWithLabel from '../molecules/IconWithLabel';
+import ProjectStatus from '../molecules/ProjectStatus';
+
+const { rem } = pixels;
+
+type ProjectDetailHeaderProps = Pick<
+  gp2Model.ProjectResponse,
+  | 'title'
+  | 'status'
+  | 'startDate'
+  | 'endDate'
+  | 'members'
+  | 'projectProposalUrl'
+> & {
+  backHref: string;
+};
+
+const rowStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  [crossQuery]: {
+    flexDirection: 'row',
+  },
+});
+
+const infoContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  gap: rem(32),
+  [drawerQuery]: {
+    display: 'unset',
+  },
+});
+
+const { getCounterString } = utils;
+
+const ProjectDetailHeader: React.FC<ProjectDetailHeaderProps> = ({
+  title,
+  status,
+  startDate,
+  endDate,
+  members,
+  projectProposalUrl,
+  backHref,
+}) => (
+  <header>
+    <BackLink href={backHref} />
+    <CardWithBackground image={projectsImage}>
+      <div css={[rowStyles, css({ gap: rem(4) })]}>
+        <div css={css({ display: 'inline-flex' })}>
+          <ProjectStatus status={status} />
+        </div>
+        {projectProposalUrl && (
+          <div
+            css={css({
+              [crossQuery]: {
+                marginLeft: 'auto',
+              },
+            })}
+          >
+            <ExternalLink
```

Comment:
This looks very similar to the ProjectCard, are they the same? Should we be extracting away to separate components and re-use them?